hotdoc/core/symbols.py

```diff
@@ -28,6 +28,8 @@ class Symbol(Base):
     display_name = Column(String)
     filename = Column(String)
     lineno = Column(Integer)
+    extent_start = Column(Integer)
```

Comment:
It does not overlap in the end because for example you have the following function:

```
static void
just_a_func (void)
{
 do = somthing()
}
```
Then:

```
FuncDecl.lineo = 2
FuncDecl.extent_start = 1
FuncDecl.extent_end = 5
```

Could be renamed, not sure what is best tbh :)